services/monitor/instances/monitor_gce_instance_service_account/constraints/no_default_service_account/constraint.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPComputeServiceAccountConstraintV1
+metadata:
+  name: no_default_service_account
+  annotations:
+    description: GCE VMs cannot use default service account.
+    category: Identity and Access Management
+    documentation: https://github.com/adeo/DataFactoryArchitecture/blob/master/Heimdall-Manual/components/gce_instance_service_account.md
```

Comment:
Could you remove line 22 with the private link please ?

Reply:
I already did